mapviz/src/mapviz.ui

```diff
@@ -350,9 +351,9 @@
     <string>Open Config</string>
    </property>
   </action>
-  <action name="actionSave_config">
+  <action name="actionSaveAs_config">
    <property name="text">
-    <string>Save Config</string>
+    <string>Save Config As</string>
```

Comment:
```suggestion
    <string>Save Config As...</string>
```

Common convention is to have menu options that open dialogs end with an ellipsis.

Reply:
Somehow I was not aware of that. Interesting